pyrlang/dist_proto/base_dist_protocol.py

```diff
@@ -246,33 +264,50 @@ def on_passthrough_message(self, control_term, msg_term):
         elif ctrl_msg_type in [CONTROL_TERM_EXIT, CONTROL_TERM_EXIT2]:
             (_, from_pid, to_pid, reason) = control_term
             if to_pid.is_local_to(n):
-                n.exit_process(from_pid, to_pid, reason)
+                n.  exit_process(from_pid, to_pid, reason)
```

Comment:
Whitespace

Reply:
wow, interesting that I missed that one.